src/component/requests.js

```diff
@@ -15,32 +17,69 @@ export default class Requests extends React.Component {
     });
   }
 
+  componentDidMount() {
+    const requests = React.findDOMNode(this);
```

Comment:
Wording: I see the variable name "requests" or some variation of it being used multiple times in that PR but often I just can't see the relation between the variable name and the data that the variable is holding.

I would rename to : scrollableDomNode or containerDomNode or listDomNode.